iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/DataRegion.java

```diff
@@ -1063,6 +1065,10 @@ private boolean insertTabletToTsFileProcessor(
       long timePartitionId) {
     // return when start >= end or all measurement failed
     if (start >= end || insertTabletNode.allMeasurementFailed()) {
+      logger.debug(
+          "Won't insert tablet {}, because {}",
+          insertTabletNode.getSearchIndex(),
+          start >= end ? "start >= end" : "insertTabletNode allMeasurementFailed");
```

Comment:
```suggestion
  if (logger.isDebugEnabled()) {
      logger.debug(
          "Won't insert tablet {}, because {}",
          insertTabletNode.getSearchIndex(),
          start >= end ? "start >= end" : "insertTabletNode allMeasurementFailed");
  }
```

Reply:
sure